releasetasks/util.py

```diff
@@ -56,6 +57,7 @@ def buildbot2bouncer(platform):
     return bouncer_platform_map.get(platform, platform)
 
 
+@retriable(sleeptime=0, jitter=0, retry_exceptions=(requests.HTTPError,))
```

Comment:
Keep in mind that with sleeptime=0 it won't help you if the service you are trying to reach has a downtime.